data/si_diffusion_v1/create_data.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+TEMPERATURE=300
+BOX_SIZE=4
+STEP=1000
+CROP=100
+NTRAIN_RUN=10
+NVALID_RUN=5
+
+NRUN=$(($NTRAIN_RUN + $NVALID_RUN))
+
+for SEED in $(seq 1 $NRUN);
+do
+  if [ "$SEED" -le $NTRAIN_RUN ]; then
+    MODE="train"
+  else
+    MODE="valid"
+  fi
+  echo $MODE $SEED
+  mkdir -p "${MODE}_run_${SEED}"
+  cd "${MODE}_run_${SEED}"
+  lmp < ../in.si.lammps -v STEP $(($STEP + $CROP)) -v T $TEMPERATURE -v S $BOX_SIZE -v SEED $SEED
```

Comment:
The `in.si.lammps` file is in the same folder as this script. The script cannot run as is and the path must be fixed.